clusterloader2/cmd/clusterloader.go

```diff
@@ -127,7 +127,13 @@ func main() {
 	for _, clusterLoaderConfig.TestConfigPath = range testConfigPaths {
 		glog.Infof("Running %v", clusterLoaderConfig.TestConfigPath)
 		if errList := test.RunTest(f, &clusterLoaderConfig); !errList.IsEmpty() {
-			glog.Fatalf("Test execution failed: %v", errList.String())
+			if metricErrList := errList.Filter(errors.IsMetricViolationError); !metricErrList.IsEmpty() {
+				glog.Errorf("Test slos failed: %v", errList.String())
```

Comment:
I'm not entirely sure this is the right approach.
What are the other errors that could be returned from here?

[In general, there aren't many errors that should block the next test, so I would like to understand cases where we don't allow next test]

Reply:
I've changed `glog.Fatal` to `glog.Error` so it will not block next tests.

TBH I don't thing that this is final version of error handling. Previous implementation blocked tests if any error was risen. I'm positive that metric violation errors shouldn't block test execution and this main change in this PR.
I added TODO, so execution errors handling will be fixed in the future.

Reply:
Does this glog error provide any value? We should have logged that already earlier anyway.

Reply:
IMHO it is like a summary of what happened. Instead of jumping through whole log, you can see type of metric violations in one place.

Reply:
So given though we are logging all errors anyway now, why not simplify all this stuff (for now) to:
```
glog.Errorf("Test execution failed: %v", errList.String())
continue
```

Reply:
That's true. I will change it.

Anyway I would like to log metric violation errors and execution errors separately - those are IMHO two different thing. I will create separate PR in future regarding this matter.